opennlp-tools/src/main/java/opennlp/tools/util/model/BaseModel.java

```diff
@@ -416,10 +416,8 @@ protected void validateArtifactMap() throws InvalidFormatException {
 
       // Version check is only performed if current version is not the dev/debug version
       if (!Version.currentVersion().equals(Version.DEV_VERSION)) {
-        // Major and minor version must match, revision might be
-        // this check allows for the use of models of n minor release behind current minor release
-        if (Version.currentVersion().getMajor() != version.getMajor() ||
-            Version.currentVersion().getMinor() - 4 > version.getMinor()) {
+        // Support OpenNLP 1.x models.
+        if (version.getMajor() != 1) {
```

Comment:
Since we're going to be releasing OpenNLP 2.0 we can't just compare the major version numbers. So replacing the check with a check to just make sure the model file is version 1.x.